ironic_tempest_plugin/tests/scenario/ironic_standalone/test_basic_ops.py

```diff
@@ -436,3 +436,18 @@ class BaremetalRedfishIPxeWholediskHttpLink(
     @utils.services('network')
     def test_ip_access_to_server(self):
         self.boot_and_verify_node()
+
+class BaremetalIdracDirectWholediskHttpLink(
+        bsm.BaremetalStandaloneScenerioTest):
+
+    api_microversion = '1.31'
```

Comment:
compare the api microversion if its correct. If its matches the release/feature version of virtual boot media.

Reply:
Checked but not found teh exact version